android/facebook-android-sdk/constants.gradle

```diff
@@ -0,0 +1,8 @@
+project.ext {
+    gradleVersion = "3.1.2"
+    buildTools = "27.0.3"
+    supportLibraryVersion = "27.0.2"
+    compileSdk = 27
+    minSdk = 15
```

Comment:
since our minsdk is 19, is this okay ?

Reply:
It is ok because it is original parameter and only higher restriction impacts on lower